WordPress/src/main/res/values/site_creation_intents.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="UnusedResources">
+    <string-array name="site_creation_intents_slugs">
+        <item>business_services</item>
+        <item>food</item>
+        <item>health</item>
+        <item>home_improvement</item>
+        <item>local_services</item>
+        <item>news</item>
+        <item>photography</item>
+        <item>travel</item>
+    </string-array>
+    <string-array name="site_creation_intents_strings">
+        <item>@string/site_creation_intents_business_services_intent</item>
+        <item>@string/site_creation_intents_food_intent</item>
+        <item>@string/site_creation_intents_health_intent</item>
+        <item>@string/site_creation_intents_home_improvement_intent</item>
+        <item>@string/site_creation_intents_local_services_intent</item>
+        <item>@string/site_creation_intents_news_intent</item>
+        <item>@string/site_creation_intents_photography_intent</item>
+        <item>@string/site_creation_intents_travel_intent</item>
+    </string-array>
+    <string-array name="site_creation_intents_emojis">
+        <item>&#128188;</item><!-- Briefcase -->
+        <item>&#127828;</item><!-- Cheeseburger -->
+        <item>&#10084;</item><!-- Heart -->
+        <item>&#127968;</item><!-- House -->
+        <item>&#8505;</item><!-- Information -->
+        <item>&#128478;</item><!-- Newspaper -->
+        <item>&#128247;</item><!-- Camera -->
+        <item>&#9992;</item><!-- Airplane -->
+    </string-array>
```

Comment:
I'm not a big fan of this pattern, having 3 "parallel arrays", since there is a possibility they can get out-of-sync. However, this seems like a reasonable approach for now, since this allows the strings for the intent descriptions to be translated in the usual way, and the number of verticals is very small. Wdyt?

Reply:
> I'm not a big fan of this pattern, having 3 "parallel arrays", since there is a possibility they can get out-of-sync.

I'm sharing the same opinion but I think the solution serves our needs best; I guess it's always a trade-off 😌